rootfs/scheduler/__init__.py

```diff
@@ -961,7 +968,7 @@ def _create_rc(self, namespace, name, image, command, **kwargs):  # noqa
 
             time.sleep(1)
 
-        return resp.json()
+        return response
```

Comment:
I can move it out into a separate PR - I did this as a part of a previous iteration of this PR